web/styles/rendered_markdown.css

```diff
@@ -716,7 +716,7 @@
         /* Having absolute positioning here ensures that the element
         doesn't scroll along with the code div in narrow windows */
         position: absolute;
-        right: 2px;
+        right: 20px;
```

Comment:
This has enough follow-ups that I think we should drop it; see 2df6b6c02a82ce36cb262b88c34970236157de37 and acf13e49cffade5b763c7639cae342eeb5817c86 as follow-ups.